tools/deploy-to-master-stable-branch.sh

```diff
@@ -40,7 +40,14 @@ fi
 echo ""
 
 echo "Building Jetpack"
-npm install -g yarn@0.16.1
+
+# Checking for yarn
+hash yarn 2>/dev/null || {
```

Comment:
It looks like the bug has already been fixed, and the nightly for yarn is already 0.18. I don't think it's worth it to require 0.16.1 anymore, do you?